src/apps/explorer/components/TransactionsTableWidget/index.tsx

```diff
@@ -81,9 +80,10 @@ export const TransactionsTableWidget: React.FC<Props> = ({ txHash }) => {
     setTabViewSelected(TabView[newTabViewName])
   }, [])
 
-  useEffect(() => {
-    history.replace({ search: `?tab=${TabView[tabViewSelected].toLowerCase()}` })
-  }, [history, tabViewSelected])
+  useEffect(
+    () => updateQueryString('tab', TabView[tabViewSelected].toLowerCase()),
+    [tabViewSelected, updateQueryString],
+  )
```

Comment:
The actual fix is to avoid replacing the fill `search` path and instead just update the `tab` key